DATA/production/qc-async/mft.json

```
@@ -46,7 +68,21 @@
         }
       }
     },
-    "checks": {}
+    "checks": {
+      "MFTClusters": {
+        "active": "true",
+        "checkName": "Clusters",
+        "dataSource": [{
+          "type": "Task",
+          "name": "Clusters",
```

```suggestion
          "name": "MFTClusters",
```
You need the task id here (the key that leads to your task config structure).